### ISSUE TITLE
Bad performance in small balloon dataset

### ISSUE THREAD

**Issue by fabro66** [hustvl/SparseInst#41]
Hi~ thanks for sharing great work!

When I train for the [balloon dataset](https://github.com/matterport/Mask_RCNN/tree/master/samples/balloon), it gets a bad performance. All the settings are adopted the same as you train SparseInst on COCO, Except `NUM_CLASSES: 1`, `MASK_FORMAT: "polygon"` and `IMS_PER_BATCH: 8`.　Are there any other hyperparameters I need to tune?

**loss_ce**: too small?
![image](https://user-images.githubusercontent.com/43380407/174546398-977a3914-d417-48ae-9e75-a28cb3906e14.png)

**Test on the train dataset**
![image](https://user-images.githubusercontent.com/43380407/174546033-0609ccde-19ad-4553-9c6b-b34e3af9c25e.png)

**Comment by fabro66** [comment]
Thank you for your prompt reply!

The balloon dataset is a very small dataset, the number of balloons in each image is around 1 to 20. There are only 61 images for training. More details can be found [here](https://github.com/matterport/Mask_RCNN/releases).

If I use COCO-pretrained model and set batch size to 16 and learning rate to 0.00001, it gets good performance.  However, If I don't use the COCO-pretrained model, it still gets bad performance. 

I am very interested in your work, will you provide a lighter backbone? For example resnet18, mobilenet.

**Comment by wondervictor** [comment]
Hi @fabro66, the balloon dataset is so small to train SparseInst (or Mask R-CNN) from scratch. We strongly suggest you for training SparseInst with the COCO-pretrained model, which has been adopted by [source of balloon dataset](https://github.com/matterport/Mask_RCNN/tree/master/samples/balloon).
We'd like to release these lighter backbones later, and we provide a mobilenetv2 model for you now: [model & config & checkpoint](https://drive.google.com/file/d/16Y8Q9jUkhINxO69LM09v_8FokYvdt2eu/view?usp=sharing)

**Comment by wondervictor** [comment]
Thanks for your interest in SparseInst 😊 . If you find it useful in your work or research, could you give us a star 🌟 or help us recommend SparseInst to your friends.

**Comment by fabro66** [comment]
Thank you very much! Help me a lot!

When training a new dataset, it is important to use a pretrained model. Looking forward to your follow-up work！
